### PR TITLE
Display risks tab-pane if risky release

### PR DESCRIPTION
**Pull request by olaurendeau**
Solve previously opened issue - https://github.com/zendesk/samson/issues/648

capture d ecran 2015-11-21 a 15 36 28

On ```Review``` button click show risk pane
capture d ecran 2015-11-21 a 15 37 01

### CODE REVIEW

**Comment by olaurendeau** [comment]
up ?

File: app/views/deploys/changeset.html.erb [hunk 1]
```diff
@@ -10,7 +10,7 @@
   <%= render 'changeset/pulls', changeset: @changeset %>
 </div>
 
-<div class="tab-pane" id="risks">
+<div class="tab-pane" id="risks" data-risky="<%= @changeset.risks? ? 1 : 0 %>">
```
Comment (steved) [inline app/views/deploys/changeset.html.erb]: How about adding this to the tab link itself? And maybe making it generic (like an `active` class or `data-active`)?

**Comment by olaurendeau** [comment]
@steved Indeed look way better I think :)

**Comment by henders** [comment]
:+1:

**Comment by steved** [comment]
Awesome, thanks!

**Comment by olaurendeau** [comment]
Thanks to you, Samson is awesome !

**Pull request merged**